modules/admin/admin-impl/src/main/java/com/enonic/xp/admin/impl/rest/resource/project/json/ProjectJson.java

```diff
@@ -38,56 +36,39 @@ public ProjectJson( final Project project, final ProjectPermissions projectPermi
             projectPermissions.getViewer().getSet() ) ) : null;
     }
 
+    @SuppressWarnings("UnusedDeclaration")
```

Comment:
Don't we have tests for Json classes?

Reply:
yes, added it dirrect checks few minutes ago, forgot to remove :)

Reply:
we have, but without using dirrect method calls.